conda/auxlib/packaging.py

```diff
@@ -73,10 +70,14 @@
 from os import getenv, listdir, remove
 from os.path import abspath, dirname, expanduser, isdir, isfile, join
 from re import compile
-from conda.auxlib.compat import shlex_split_unicode
 from subprocess import CalledProcessError, PIPE, Popen
 import sys
 
+from .compat import shlex_split_unicode
+from ..deprecations import deprecated
+
+deprecated.module("23.9", "24.3", addendum="Upgrade your build system to conform with PEP-517.")
```

Comment:
Should this be "use https://pypi.org/project/packaging/ instead"?

Reply:
I'm not sure how the user would follow this advice or know what PEP-517 is. And why my current build system would import from conda.auxlib.packaging? It could read "No longer used by conda"?